test-case/verify-sof-firmware-load.sh

```diff
@@ -22,6 +22,13 @@ func_opt_parse_option "$@"
 
 setup_kernel_check_point
 
+( set +x
```

Comment:
Left over debug code?

Reply:
i copy-pasted it from your SOF_LOGGING work :-)

I have no idea what this does.